src/main/scala/geekie/mapreddemo/AkkaMapreduceBenchmark.scala

```diff
@@ -1,62 +1,40 @@
 package geekie.mapreddemo
```

Comment:
@rbaron I want the examples to be easily separated from the actual library, but I'm still unsure what is the best way. Maybe we should create a folder next to `/src`, just like `/test`, and then it would be fine to have the test code in that package inside `geekie.mapred`. But with the code sharing the same `/src` root, I think it's best to keep it in a different package... Or maybe we could turn these aps into actual tests?